gma/src/common/gma_common.cc

```diff
@@ -95,6 +97,19 @@ void GmaInternal::UpdateAdViewInternalAdSizeDimensions(
   ad_view_internal->update_ad_size_dimensions(width, height);
 }
 
+void GmaInternal::InsertNativeInternalImage(
+    internal::NativeAdInternal* native_ad_internal,
+    const NativeAdImageInternal& native_image_internal, std::string image_type,
+    bool clear_existing_images) {
+  assert(native_ad_internal);
+
+  if (clear_existing_images) {
+    native_ad_internal->clear_existing_images();
+  }
+  NativeAdImage icon_image = NativeAdImage(native_image_internal);
```

Comment:
Should this be just named image instead of icon_image since you are taking that as a input, right? Also the parameter should probably be const std::string& image_type.

Reply:
Done.